src/js/utils/events.js

```diff
@@ -53,7 +53,7 @@ export function on(elem, type, fn){
           if (event.isImmediatePropagationStopped()) {
             break;
           } else {
-            handlersCopy[m].call(elem, event);
+            handlersCopy[m].apply(elem, [event].concat(evtData));
```

Comment:
@heff I did update them 3 days ago. I don't know why they are not shown here. 
Can you please take a look a the changes in https://github.com/videojs/video.js/compare/master...carpasse:feature/event-emitter-enhance